### PR TITLE
Add plugin 'metalsmith-hooks' to plugins directory

### PR DESCRIPTION
**Pull request by benwilhelm**

### CODE REVIEW

File: src/plugins.json [hunk 1]
```diff
@@ -493,6 +493,11 @@
     "description": "Add an id and anchor to all headings on a page. Ideal for permalinks."
   },
   {
+    "name": "Hooks",
+    "repository": "https://github.com/benwilhelm/metalsmith-hooks",
```
Comment (Ajedi32) [inline src/plugins.json]: This is missing an "icon" attribute. (If you're not sure what to put, check out the other plugins on metalsmith.io and copy one of the icons from those.)

**Comment by benwilhelm** [comment]
@Ajedi32 - Updated with an icon per your request.

File: src/plugins.json [hunk 2]
```diff
@@ -493,6 +493,12 @@
     "description": "Add an id and anchor to all headings on a page. Ideal for permalinks."
   },
   {
+    "name": "Hooks",
+    "icon": "hook",
```
Comment (Ajedi32) [inline src/plugins.json]: That's great. Unfortunately I'm pretty sure 'hook' isn't one of the available icons. (At least, I didn't see it mentioned in the symbolset CSS file and no other plugin on the list has this as their icon.)

Try looking over the existing list of plugins and copy one of the icons from there. Or alternately, check the [symbolset css file](http://cdn.symbolset.com/94440cd0456394a198fb9c3302ad80fe007e5eff/symbolset.css) for an icon that fits your plugin and use that.

Comment (benwilhelm) [reply src/plugins.json]: Whoops! Disconnect between brain and fingers.  That should have been "fork".  Wrong metal implement :)

**Pull request merged**